### PR TITLE
Provide a more precise install guide for Archlinux

### PR DESCRIPTION
**Pull request by SamantazFox**
Precise clearly how to install the crystal version required by invidious on Archlinux.

### CODE REVIEW

**Comment by unixfox** [comment]
I would invert the download of the package from an URL solution and the "downgrade" package solution because IMO the "downgrade" package is much easier to use and compatible with archlinux directly.
The download of the package from an URL is IMO only suitable for forks of ArchLinux like Manjaro.

**Review by unixfox — APPROVED** [review]

File: Installation.md [hunk 1]
```diff
@@ -132,20 +132,23 @@ Arch Linux
...
-# Invidious depends on crystal 0.36.1-1, so it has to be installed from the archive.
+# Invidious depends on crystal 0.36.1-1, so you either have to downgrade Crystal or install it from the archive.
+
+# Downgrade using the `downgrade` script from the AUR ( https://aur.archlinux.org/packages/downgrade/ ):
+# A basic understanding of the AUR is required, and the installation of `yay` (or `paru`) has to be done
+sudo pacman -S crystal
+yay -S downgrade (or `paru -S downgrade`)
+sudo downgrade crystal
+# > Select 0.36.1-1
+# When asked if you want to add it to the ignore list (so that it's never updated), you can choose yes.
```
Comment (SamantazFox) [inline Installation.md:143]: I think that saying the following could be more effective:
`When asked to "add crystal to the ignore list" (so it won't be updated in the future), choose "yes".`

**Comment by SamantazFox** [comment]
> I would invert the download of the package from an URL solution and the "downgrade" package solution because IMO the "downgrade" package is much easier to use and compatible with archlinux directly.
> The download of the package from an URL is IMO only suitable for forks of ArchLinux like Manjaro.

Mmmmh, sure. I'm not found of using the AUR, but I'm gonna follow your advices as you and @TheFrenchGhosty are more experienced with Arch than me.

**Comment by TheFrenchGhosty** [comment]
@SamantazFox AUR packages are recommended a lot even in the Arch wiki, it's an amazing tool, and there's really no reason not to use it.

**Pull request merged**